mlflow/store/model_registry/file_store.py

```diff
@@ -0,0 +1,769 @@
+            )
+        return self._get_registered_model_from_path(model_path)
+
+    def get_latest_versions(self, name, stages=None):
+        """
+        Latest version models for each requested stage. If no ``stages`` argument is provided,
+        returns the latest version for each stage.
+
+        :param name: Registered model name.
+        :param stages: List of desired stages. If input list is None, return latest versions for
+                       each stage.
+        :return: List of :py:class:`mlflow.entities.model_registry.ModelVersion` objects.
+        """
+        registered_model_path = self._get_registered_model_path(name)
+        if registered_model_path is None:
+            raise MlflowException(
+                f"Could not find registered model with name {name}",
+                RESOURCE_DOES_NOT_EXIST,
+            )
+        model_versions = self._list_model_versions_under_path(registered_model_path)
+        if stages is None or len(stages) == 0:
+            expected_stages = {get_canonical_stage(stage) for stage in ALL_STAGES}
+        else:
+            expected_stages = {get_canonical_stage(stage) for stage in stages}
+        latest_versions = {}
+        for mv in model_versions:
+            if mv.current_stage in expected_stages:
+                if (
+                    mv.current_stage not in latest_versions
+                    or latest_versions[mv.current_stage].version < mv.version
+                ):
+                    latest_versions[mv.current_stage] = mv
+
+        return [latest_versions[stage] for stage in expected_stages if stage in latest_versions]
+
+    def _get_registered_model_tag_path(self, name, tag_name):
+        _validate_model_name(name)
+        _validate_tag_name(tag_name)
+        registered_model_path = self._get_registered_model_path(name)
+        if registered_model_path is None:
+            raise MlflowException(
+                f"Could not find registered model with name {name}",
+                RESOURCE_DOES_NOT_EXIST,
+            )
+        return os.path.join(registered_model_path, FileStore.TAGS_FOLDER_NAME, tag_name)
+
+    def _get_registered_model_tag_from_file(self, parent_path, tag_name):
+        _validate_tag_name(tag_name)
+        tag_data = read_file(parent_path, tag_name)
+        return RegisteredModelTag(tag_name, tag_data)
+
+    def _get_resource_files(self, root_dir, subfolder_name):
+        source_dirs = find(root_dir, subfolder_name, full_path=True)
+        if len(source_dirs) == 0:
+            return root_dir, []
+        file_names = []
+        for root, _, files in os.walk(source_dirs[0]):
+            for name in files:
+                abspath = join(root, name)
+                file_names.append(os.path.relpath(abspath, source_dirs[0]))
+        if sys.platform == "win32":
+            # Turn metric relative path into metric name.
+            # Metrics can have '/' in the name. On windows, '/' is interpreted as a separator.
```

Comment:
Nit: Can registered models / model versions contain the `/` name? Perhaps we want to update the comment here to refer to registered models & model versions instead of metrics

Reply:
I don't see any checks preventing '/' in the model name so we can keep this. I'll update the comment, thanks for catching it!